migrations/versions/dcecb0cd0cea_.py

```diff
@@ -0,0 +1,50 @@
+"""empty message
```

Comment:
Any reason you added two migrations rather than one?

Reply:
No special reason.  We will merge our database updates down the line once we have a more stable version of the backend.

Reply:
I have merged the database updates into one migration